### PR TITLE
pd-control: add description for `scheduler config evict-leader-scheduler`

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

Add a description for `scheduler config evict-leader-scheduler` in the PD control doc.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/17755

### CODE REVIEW

File: pd-control.md [hunk 1]
```diff
@@ -804,6 +804,8 @@ Usage:
 >> scheduler add grant-leader-scheduler 1         // Schedule all the leaders of the Regions on store 1 to store 1
 >> scheduler add evict-leader-scheduler 1         // Move all the Region leaders on store 1 out
 >> scheduler config evict-leader-scheduler        // Display the stores in which the scheduler is located since v4.0.0
+>> scheduler config evict-leader-scheduler add-store 2    // Add leader eviction scheduling for store 2
```
Comment (JmPotato) [inline pd-control.md:807]: Do we need to align the comment?

Comment (Oreoxmt) [reply pd-control.md]: https://github.com/pingcap/docs/pull/17968/commits/7bd5ada8b2dacec88c838515aabc4f0c67bca643

**Review by JmPotato — APPROVED** [review]

File: pd-control.md [hunk 2]
```diff
@@ -800,20 +800,22 @@ Use this command to view and control the scheduling policy.
...
->> scheduler config balance-hot-region-scheduler  // Display the configuration of the balance-hot-region scheduler
->> scheduler describe balance-region-scheduler    // Display the running state and related diagnostic information of the balance-region scheduler
+>> scheduler show                                          // Display all created schedulers
+>> scheduler add grant-leader-scheduler 1                  // Schedule all the leaders of the Regions on store 1 to store 1
+>> scheduler add evict-leader-scheduler 1                  // Move all the Region leaders on store 1 out
+>> scheduler config evict-leader-scheduler                 // Display the stores in which the scheduler is located since v4.0.0
+>> scheduler config evict-leader-scheduler add-store 2     // Add leader eviction scheduling for store 2
+>> scheduler config evict-leader-scheduler delete-store 2  // Remove leader eviction scheduling for store 2
+>> scheduler add shuffle-leader-scheduler                  // Randomly exchange the leader on different stores
+>> scheduler add shuffle-region-scheduler                  // Randomly scheduling the Regions on different stores
```
Comment (hfxsd) [inline pd-control.md:810]:
```suggestion
>> scheduler add shuffle-region-scheduler                  // Randomly schedule the Regions on different stores
```

**Review by hfxsd — APPROVED** [review]

**Pull request merged**